generators/generator-base.js

```diff
@@ -871,7 +871,7 @@ Generator.prototype.copyTemplate = function (source, dest, action, _this, _opt,
             jhipsterUtils.copyWebResource(source, dest, regex, 'html', _this, _opt, template);
             break;
         case 'stripJs' :
-            var regex = '[a-zA-Z]+\:(\s)?\[[ \'a-zA-Z0-9\$\,\(\)\{\}\n\.\<\%\=\>\;\s]*\}\]';
+            var regex = '\,(\s)*(resolve)+\:(\s)?[\{](\s)*[a-zA-Z]+\:(\s)?\[[ \'a-zA-Z0-9\$\,\(\)\{\}\n\.\<\%\=\-\>\;\s]*\}\](\s)*\}';
```

Comment:
yes this will remove the entire resolve block and we dont have anything else in the resolve block currently so this is fine, if we add anything to resolve in future then we need to amend this. Currently it was creating empty resolve blocks which was ugly